src/components/OSCALLoader.js

```diff
@@ -233,6 +235,19 @@ export default function OSCALLoader(props) {
     }
   }, [props.isRestMode]);
 
+  // Handle anchor link change in window url
+  useEffect(() => {
+    // TODO: Handle GET, when the hash hasn't changed & initial navigation to page.
+    // This outputs a 404 error at this point in the console & doesn't take the user
+    // to the desired section of page based on hash.
+    const { hash } = window.location;
+    // Determine a hash exists and grab it
+    const hashStr = hash && document.getElementById(hash.substring(1));
+    if (hashStr) {
+      hashStr.scrollIntoView({ behavior: "smooth" });
```

Comment:
Can we maybe call this something like `elementToNavigateTo` (that's probably too long) or something like that? `hashStr` doesn't really explain what this is correctly. Even like `matchingElement` might work.